manifest.celery.production.yml

```diff
@@ -0,0 +1,13 @@
+command: ./scripts/cf_run_celery_app.sh
+buildpack: python_buildpack
+health-check-type: process
+instances: 1
+memory: 1G
+disk_quota: 512M
+services:
+- pgdb-96-prod
```

Comment:
does it need this?

Reply:
Not for emailing but if future tasks need to do any DB work then I would think so.

Reply:
actually it probably does now - it bootstraps flask which may need the DB?